system-int-test/test/cmr/system_int_test/ingest/granule_bulk_update/granule_bulk_update_schema_test.clj

```diff
@@ -0,0 +1,79 @@
+(ns cmr.system-int-test.ingest.granule-bulk-update.granule-bulk-update-schema-test
+  (:require
+   [cheshire.core :as json]
+   [clojure.string :as string]
+   [clojure.test :refer :all]
+   [cmr.common.util :as util :refer [are3]]
+   [cmr.message-queue.test.queue-broker-side-api :as qb-side-api]
+   [cmr.mock-echo.client.echo-util :as echo-util]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as sys]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(def base-request {:operation "UPDATE_FIELD"
+                   :update-field "OPeNDAPLink"
+                   :updates
+                   [["ur_1" "https://aws.foo.com"]
+                    ["ur_2" "https://aws.bar.com"]]})
+
+(deftest bulk-granule-schema-validation-test
+  (are3
+   [mutate err-msg]
```

Comment:
nit: This mutate way of doing the test cases is too verbose and hard to follow. I think if we just list the updates as the argument, it would be easier to follow.

Reply:
the operations and update-field validations can be separated out.